static/app/components/modals/importDashboardFromFileModal.tsx

```diff
@@ -97,7 +97,6 @@ function ImportDashboardFromFileModal({
         <Wrapper>
           <Button
             onClick={handleUploadClick}
-            size="md"
```

Comment:
was this an intentional change? (not icon related)

Reply:
Yea. `md` is the default.

Sorry I kind of snuck it in